package.json

```diff
@@ -10,7 +10,9 @@
     "build": "rm -rf ./client/dist/ && NODE_ENV=production webpack",
     "watch": "webpack --watch --progress --colors -d",
     "analyze": "rm -rf ./client/dist/ && BUNDLE_ANALYZER=on webpack",
-    "analyze:build": "rm -rf ./client/dist/ && NODE_ENV=production BUNDLE_ANALYZER=on webpack"
+    "analyze:build": "rm -rf ./client/dist/ && NODE_ENV=production BUNDLE_ANALYZER=on webpack",
```

Comment:
I notice there's lots of repetition in calling `rm -rf ./client/dist`, could we make this a `clean` step and use pre steps to trigger them?

From https://docs.npmjs.com/misc/scripts:

> Pre and post commands with matching names will be run for those as well (e.g. premyscript, myscript, postmyscript).

Reply:
The pre/post steps are action specific, so I'm not sure how can they save us any repetition  here. 

But we can define the `rm -rf` part as a `clean` command, just to avoid typos in the future.

Reply:
Cool, makes sense!